dominos/src/main/java/me/fontys/semester4/dominos/ImportAll.java

```diff
@@ -0,0 +1,97 @@
+package me.fontys.semester4.dominos;
+
+import me.fontys.semester4.data.entity.PostalcodePart;
+import me.fontys.semester4.data.entity.Product;
+import me.fontys.semester4.data.entity.Store;
+import me.fontys.semester4.data.repository.*;
+import me.fontys.semester4.dominos.pizza.and.ingredient.PizzaAndIngredientConverter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+import java.io.FileNotFoundException;
+import java.math.BigDecimal;
+
+@SpringBootApplication
+@EntityScan(basePackages = "me.fontys.semester4")
+@EnableJpaRepositories(basePackages = "me.fontys.semester4")
+public class ImportAll implements CommandLineRunner {
+
+    private final StoreRepository storeRepository;
+    private final IngredientRepository ingredientRepository;
+    private final CategoryRepository categoryRepository;
+    private final ProductPriceRepository productPriceRepository;
+    private final ProductRepository productRepository;
+    private final PostalcodePartRepository postalcodePartRepository;
+    private final PizzaAndIngredientConverter pizzaAndIngredientConverter;
+
+    public static void main(String[] args) {
+        SpringApplication.run(ImportAll.class, args);
+    }
+
+
+    @Autowired
+    public ImportAll(StoreRepository storeRepository,
+                     ProductRepository productRepository,
```

Comment:
Please see my PR (merged by @mdevrees already into master) on how we can best use the main file. If we all start adding individual dependencies and logic in the main file, it will become extremely cluttered very quickly. Any logic here in the main file is also logic you can't invoke through a test case; so this the main file should have minimal changes (think of it like a controller); we should only invoke logic here, not have logic.

Reply:
Agreed. Meant to be temp. I made my own importers. Push inc.